src/create-model.jl

```diff
@@ -80,20 +75,9 @@ function create_model(
 
     ## Add expressions to dataframes
     # TODO: What will improve this? Variables (#884)?, Constraints?
-    (
-        incoming_flow_lowest_resolution,
-        outgoing_flow_lowest_resolution,
-        incoming_flow_lowest_storage_resolution_intra_rp,
-        outgoing_flow_lowest_storage_resolution_intra_rp,
-        incoming_flow_highest_in_out_resolution,
-        outgoing_flow_highest_in_out_resolution,
-        incoming_flow_highest_in_resolution,
-        outgoing_flow_highest_out_resolution,
-        incoming_flow_storage_inter_rp_balance,
-        outgoing_flow_storage_inter_rp_balance,
-    ) = add_expressions_to_dataframe!(
-        dataframes,
+    add_expressions_to_dataframe!(
```

Comment:
Maybe we need to rename this function —something like "add_expressions_for_constraints" to be more specific.

Reply:
Great! Thanks